pkgs/by-name/be/beads/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

Suggestion because of #315337